tasks/push-snapshot/README.md

```diff
@@ -4,16 +4,19 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 
 ## Parameters
 
-| Name            | Description                                                                                     | Optional | Default value        |
-|-----------------|-------------------------------------------------------------------------------------------------|----------|----------------------|
-| snapshotPath    | Path to the JSON string of the mapped Snapshot spec in the data workspace                       | Yes      | mapped_snapshot.json |
-| tag             | Default tag to use if mapping entry does not contain a tag. Not used when tagPrefix is set      | Yes      | latest               |
-| tagPrefix       | The prefix used to build the tag in <prefix>-<timestamp> format                                 | Yes      | ""                   |
-| timestampFormat | Timestamp format used to build the tag when tagPrefix is set                                    | Yes      | %s                   |
-| retries         | Retry copy N times                                                                              | Yes      | 0                    |
-| addGitShaTag    | Also push a tag with the git sha for each image in the Snapshot                                 | Yes      | true                 |
-| addSourceShaTag | Also push a tag with the source sha for each image in the Snapshot                              | Yes      | true                 |
-| addTimestampTag | Also push a tag with the current timestamp for each image in the Snapshot                       | Yes      | false                |
+| Name         | Description                                                               | Optional | Default value        |
+|--------------|---------------------------------------------------------------------------|----------|----------------------|
+| snapshotPath | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes      | mapped_snapshot.json |
+| dataPath     | Path to the JSON string of the merged data to use in the data workspace   | Yes      | data.json            |
+| retries      | Retry copy N times                                                        | Yes      | 0                    |
```

Comment:
small question: why is this not in `data`?

Reply:
It probably should be, but I couldn't find any RPAs using this so I left it for now and focused on the high priority things. I can change it though, probably best to do it all at once

Reply:
we can tackle in a subsequent PR

Reply:
It actually isn't a pipeline parameter for any pipeline so can't be configured right now